pwiz_tools/Skyline/TestFunctional/FullScanGraphTest.cs

```diff
@@ -193,7 +193,9 @@ protected override void DoTest()
             Assert.AreEqual(testRange, SkylineWindow.GraphFullScan.Range);
 
             RunUI(() => SkylineWindow.SynchMzScale(SkylineWindow.GraphSpectrum, false)); // Sync from the library match to the full scan viewer
-            TestSpecialIonsAnnotations();
+            //annotations are not shown in the offscreen mode
+            if (!Skyline.Program.SkylineOffscreen)

```

Comment:
This seems overly drastic. If it is a painting issue, then you should add a method that tells what would be annotated during a paint, and use that to make sure the graph understands what it is supposed to do. We use this approach a lot with our graphs.